model/admin/db.go

```diff
@@ -0,0 +1,93 @@
+package admin
+
+import (
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/evergreen-ci/evergreen/db/bsonutil"
+	"github.com/pkg/errors"
+	"gopkg.in/mgo.v2/bson"
+)
+
+const (
+	Collection       = "admin"
+	systemSettingsID = "0"
+)
+
+var (
+	IdKey     = bsonutil.MustHaveTag(AdminSettings{}, "Id")
+	BannerKey = bsonutil.MustHaveTag(AdminSettings{}, "Banner")
+
+	ServiceFlagsKey  = bsonutil.MustHaveTag(AdminSettings{}, "ServiceFlags")
+	TaskDispatchKey  = bsonutil.MustHaveTag(ServiceFlags{}, "TaskDispatchDisabled")
+	HostinitKey      = bsonutil.MustHaveTag(ServiceFlags{}, "HostinitDisabled")
+	MonitorKey       = bsonutil.MustHaveTag(ServiceFlags{}, "MonitorDisabled")
+	NotificationsKey = bsonutil.MustHaveTag(ServiceFlags{}, "NotificationsDisabled")
+	AlertsKey        = bsonutil.MustHaveTag(ServiceFlags{}, "AlertsDisabled")
+	TaskrunnerKey    = bsonutil.MustHaveTag(ServiceFlags{}, "TaskrunnerDisabled")
+	RepotrackerKey   = bsonutil.MustHaveTag(ServiceFlags{}, "RepotrackerDisabled")
+	SchedulerKey     = bsonutil.MustHaveTag(ServiceFlags{}, "SchedulerDisabled")
```

Comment:
Is there a way that we can make the key names private? I don't like the key name paradigm in the other model packages: it means we basically export *everything* and I don't think that's a good pattern to follow.